lib/castle_devise/configuration.rb

```diff
@@ -39,6 +39,14 @@ class Configuration
     #   @return [Array<Proc>] Array of procs that will get called after a request to the Castle API
     config_accessor(:after_request_hooks) { [] }
 
+    # @!attribute castle_sdk_facade
+    #   @return [Class] Castle API implementation
+    config_accessor(:castle_sdk_facade) { ::CastleDevise::SdkFacade }
```

Comment:
I think we should call this setting `castle_sdk_facade_class`. Right now it doesn't seem consistent with `castle_client`, since the Client isn't instantiated but the Facade is